articles/virtual-desktop/create-host-pools-user-profile.md

```diff
@@ -14,7 +14,7 @@ manager: lizross
 
 The Windows Virtual Desktop service offers FSLogix profile containers as the recommended user profile solution. We don't recommend using the User Profile Disk (UPD) solution, which will be deprecated in future versions of Windows Virtual Desktop.
 
-This article will tell you how to set up a FSLogix profile container share for a host pool using a virtual machine-based file share. For more FSLogix documentation, see the [FSLogix site](https://docs.fslogix.com/).
+This article will tell you how to set up a FSLogix profile container share for a host pool using a virtual machine-based file share. Please note we recommend usign Azure File instead of file shares. For more FSLogix documentation, see the [FSLogix site](https://docs.fslogix.com/).
```

Comment:
```suggestion
This article will tell you how to set up a FSLogix profile container share for a host pool using a virtual machine-based file share. We strongly recommend using Azure Files instead of file shares. For more FSLogix documentation, see the [FSLogix site](https://docs.fslogix.com/).
```